src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static class Task {
```

Comment:
Consider abstracting Task, Todo, Deadline and Event to separate files rather than nesting them as a static class.

This allows for better encapsulation of functionality and data to the relevant classes, improving user readability.